nasa/cumulus-dashboard
# Cumulus 1502

This adds a copy button to an individual collection page.
Adjusted some css on the buttons to display it properly.
Also, adding in redux-logger for easier keeping track of the redux store.

To test:
1. Go to a collection
2. Click copy
3. Verify you are taken to /collection/add with the info from the collection
4. Change the version or name
5. Submit
6. Verify new collection is made
